### ISSUE TITLE
Thank for your code.. but load_model path is wrong ?

### ISSUE THREAD

**Issue by keunsoo74** [nicknochnack/ImageClassification#4]
Hi, nicknochnack

I tried clone and run your code.
I have Intel Mac Pro <2019> then I have no GPU. So, I edit your code for install libs and GPU option

!pip install tensorflow opencv-python matplotlib **sentencepiece** **keras**
!pip install torch transformers text2sql

tf.config.list_physical_devices('CPU')
#tf.config.list_physical_devices('GPU')

**AS-IS : error**
new_model = load_model('imageclassifier.h5')

**TO-BE**
new_model = load_model('models/imageclassifier.h5')

Is it right?

Thanks

**Comment by Shivansh20128** [comment]
Yes, @keunsoo74, the model path needs to be modified. The correct path is the one you have written in your code.
@nicknochnack you can close this issue now.
